### PR TITLE
[Added] UI Changes,Function Changes

### PR DESCRIPTION
**Pull request by rohandol112**
Description
1.Added different colours in button of dark and light theme.Make it properly aligned and responsive.
2.Changes in functions.
3.Added Responsiveness

Able to merge working fine.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -99,3 +99,6 @@ <h3>How to Play?</h3>
 </html>
 
 <script src="./app.js"></script>
+<audio id="moveSound" src="J:\Open Source\hactoberfest2\2048\2048\assets\320181__dland__hint.wav"></audio>
+<audio id="mergeSound" src="J:\Open Source\hactoberfest2\2048\2048\assets\75337__neotone__chime2.wav"></audio>
```
Comment (Alitindrawan24) [inline index.html:103]: is the audio still works ? Because the src is in your local directory

**Comment by rohandol112** [comment]
no i have deleted the files cuz it has a problem. i'll work on as a second PR.
is it okay?

**Comment by Alitindrawan24** [comment]
> no i have deleted the files cuz it has a problem. i'll work on as a second PR. is it okay? @Alitindrawan24 ?

Just create a new commit on this PR to delete the audio. its fine

File: index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
     />
 
     <link rel="stylesheet" href="./style.css" />
-  </head>
```
Comment (Alitindrawan24) [inline index.html:20]: This also deleted the end tag of head

**Comment by rohandol112** [comment]
ooh okay so sry @Alitindrawan24 i'm bothering too much actually im kinda beginner in development.

**Comment by rohandol112** [comment]
Thankyou for support and i guess all is fine its able to merge.

**Pull request merged**